clerk-frontend/src/App.html

```diff
@@ -0,0 +1,141 @@
+<html>
```

Comment:
What is the purpose of this file? It seems a lot of duplication with index.html in the public folder, and when I deleted it everything seems fine.